itests/org.openhab.core.thing.tests/src/main/java/org/eclipse/smarthome/core/thing/type/ChannelStateDescriptionProviderOSGiTest.java

```diff
@@ -77,14 +81,20 @@
  */
 public class ChannelStateDescriptionProviderOSGiTest extends JavaOSGiTest {
 
+    private static final String TEST_BUNDLE_NAME = "thingStatusInfoI18nTest.bundle";
+    private static final ChannelTypeUID CHANNEL_TYPE_7_UID = new ChannelTypeUID("hue:num-dynamic");
+
+    private ThingStatusInfoI18nLocalizationService thingStatusInfoI18nLocalizationService;
     private ItemRegistry itemRegistry;
     private ItemChannelLinkRegistry linkRegistry;
 
     @Mock
     private ComponentContext componentContext;
 
+    private Bundle testBundle;
+
     @Before
-    public void setup() {
+    public void setup() throws Exception {
```

Comment:
That's very generic ;)

Reply:
Unfortunately, I have to

https://github.com/openhab/openhab-core/blob/00edae4539f9ea4188d34afb68424ac8cfb33232/bundles/org.openhab.core.test/src/main/java/org/eclipse/smarthome/test/SyntheticBundleInstaller.java#L79-L92